dotnet-3.1/src/Index.cs

```diff
@@ -10,6 +10,8 @@
   
   If an error is thrown, a response with code 500 will be returned.
 */
+using System.Threading.Tasks;
```

Comment:
```suggestion

using System.Threading.Tasks;
```

Just a little extra addition